srcpkgs/guile/template

```diff
@@ -29,6 +39,17 @@ post_extract() {
 	rm -rf prebuilt/32-bit-big-endian
 }
 
+pre_configure() {
```

Comment:
`post_patch`

Reply:
> post_patch

Could you please elaborate?

Reply:
The operation should happen in `post_patch`, not `pre_configure`.

Reply:
Updated.